app/analytics.go

```diff
@@ -103,6 +103,13 @@ func (a *App) GetAnalytics(name string, teamId string) (model.AnalyticsRows, *mo
 			close(monthlyActiveChan)
 		}()
 
+		userRegisteredChan := make(chan store.StoreResult, 1)
```

Comment:
You are requesting the users, but you are not using it anywhere here

Reply:
yes, this needs to be removed since we are using the existing definition of registered users.